#163384443 adjust UI pages style

Jesse-efe opened the pull request:
#### What does this PR do?
Make adjustments to UI pages
#### Description of Task to be completed?
Add nav bar to sign-in, sign-up, and reset-password pages
Remove empty space above footer in homepage
Create a new page that shows all parties
#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#163384443
#### Screenshots
![one](https://user-images.githubusercontent.com/35180482/51599330-9eba1600-1eff-11e9-89a1-29543618510d.JPG)
![two](https://user-images.githubusercontent.com/35180482/51600602-a3cc9480-1f02-11e9-8f37-05ad8375cf66.JPG)
![three](https://user-images.githubusercontent.com/35180482/51599357-abd70500-1eff-11e9-806d-8343fb48b900.JPG)
![five](https://user-images.githubusercontent.com/35180482/51604137-23129600-1f0c-11e9-916e-4bbaa882d189.JPG)
![four](https://user-images.githubusercontent.com/35180482/51603296-b9918800-1f09-11e9-889a-07221dfce1f8.JPG)




#### Questions:
N/A

Jesse-efe committed:
Create the home page Finishes [Delivers #163312850]

Jesse-efe committed:
Created sign in page [Delivers #16329572]

Jesse-efe committed:
Create user sign up page [Delivers #163295274]

Jesse-efe committed:
Create Password reset page [Delivers #163296269]

Jesse-efe committed:
feature(createUserPageThatShowsAllPoliticalParties):

- page has a nav bar
- page has cards for each party
- page has a footer

[delivers #163351595]

Jesse-efe committed:
feat(design/template): make changes to UI

-add a navbar to sign-up, sign-in and reset password pages

[delivers #163384443]

megame24 commented:
Kindly follow the instructions on this Github page(https://github.com/andela/workdey/wiki/PR-Format) to format your PR. Reply with `N/A` to PR description question(s) that does not apply to this PR.
Also, on the same page, take a look at the right to see the commit message convention, incorporate the convention and update your commit message.

megame24 commented:
Since this task has to do with the UI, please include screenshots to the PR description.

Jesse-efe commented:
Thank you @megame24. I have implemented the changes

megame24 commented:
Let's find another color combination for the yellow sign up button, I can barely see the text in the button.

Jesse-efe commented:
Ok Let me see

Jesse-efe committed:
feat(design/template): make changes to nav bar button

-change the color of the text in the button

[delivers #163418388]

megame24 commented:
Your create account form has a black button labeled `sign in`, let's keep the color of the buttons consistent, also, update the name on the button to reflect the appropriate form.

Jesse-efe commented:
Ok I will do so now

Jesse-efe committed:
feat(design/template): change text in sign-up page submit button

-change the text from sign-in to sign-up

[delivers #163419537]

megame24 commented on UI/styles/parties.css:
```diff
@@ -0,0 +1,196 @@
+    #container > .description{
+        grid-template-columns: 1fr;
+    }
+}  */
```
Get rid of commented block of code

Jesse-efe replied:
Ok

megame24 reviewed (approved):
Implement the last feedback and merge. Good job.

Jesse-efe committed:
ch(design/template): remove commented out code in css files

[finishes #163419946]

megame24 commented:
Let's maintain a consistent color for both the primary button(blue) and the secondary button(yellow). What do think about changing the green buttons to the blue ones(primary button)?

Jesse-efe commented:
Ok I would try that

Jesse-efe committed:
feat(design/template): change color of buttons in cards

-change the color of buttons in cards in parties.html file

[delivers #163420400]

Jesse-efe committed:
feat(design/template): change color of button in home page

-change the color of call to action button

[delivers #163420702]

megame24 commented:
This is ready for merge, good job

Jesse-efe commented:
Thank you
